python/cudf/cudf/tests/test_orc.py

```diff
@@ -1696,7 +1607,7 @@ def test_orc_writer_rle_stream_size(datadir, tmpdir):
 
     # Segfaults when RLE stream sizes don't account for varint length
     pa_out = pa.orc.ORCFile(reencoded).read()
-    assert_eq(df.to_pandas(), pa_out)
+    assert df.to_arrow().equals(pa_out)
```

Comment:
looks like this was a different kind of redundant casting?

Reply:
Yup. We were going to pandas for comparison previously. Now we just do arrow level comparison.